Eva Functions:Fatima

```diff
@@ -0,0 +1,83 @@
+// Write a function calculateCube that takes a single number and prints the volume of a cube made from that number.
+ function calculateCube(number) {
+    return number ** 3;
+}
+console.log(calculateCube(5));
+
+// Write a function isAVowel that takes a character (i.e. a string of length 1) and returns true if it is a vowel, false otherwise. The vowel could be upper or lower case.
+function isAVowel(character) {
+  const vowels = ["a", "e", "i", "o", "u"];
+  if (vowels.includes(character.toLowerCase())) {
+    return true;
+  } else {
+    return false;
+  }
+
+}
+console.log(isAVowel("a"));
+
+// Write a function getTwoLengths that accepts two parameters (strings). The function should return an array of numbers where each number is the length of the corresponding string.
+function getTwoLengths(par1, par2) {
+  return [par1.length, par2.length];
+  
+}
+console.log(getTwoLengths("Hank", "Hippopopalous"));
+
+// Write a function getMultipleLengths that accepts a single parameter as an argument: an array of strings. The function should return an array of numbers where each number is the length of the corresponding string.
+function getMultipleLengths(strArray) {
+  const lengthsArray = [];
+  for (let i = 0; i < strArray.length; i++) {
+    lengthsArray.push(strArray[i].length);
+  } 
+  return lengthsArray;
+}
+console.log(getMultipleLengths(["hello", "what", "is", "up", "dude"]));
+
+
+// Define a function maxOfThree that takes three numbers as arguments and returns the largest of them. If all numbers are the same, it doesn't matter which one is returned. If the two largest numbers are the same, one of them should be returned.
+
+// Write a function printLongestWord that accepts a single argument, an array of strings. The method should return the longest word in the array. In case of a tie, the method should return the word that appears first in the array.
+
+function printLongestWord(strArray) {
+  for (let i =0; i < arr.length; i++) {
+  let longestWord = arr[0]
+  if (arr[i].length > longestWord.length) {
+  longestWord === arr[i]
+  }
+  }
```

Comment:
careful with your indentation here! lines 43-46 should be indented one level, and line 45 should be indented one extra level. your formatting has been flawless up till here, though, so great work so far!